cmd/kpod/parse.go

```diff
@@ -0,0 +1,872 @@
+// most of these validate and parse functions have been taken from projectatomic/docker
```

Comment:
Is this from @umohnani8 and if so does it require separate review here?

Reply:
yes